backend/schema/validate.go

```diff
@@ -512,9 +512,16 @@ func validateIngressRequestOrResponse(scopes Scopes, n *Verb, reqOrResp string,
 	if opt, ok := fieldType.(*Optional); ok {
 		fieldType = opt.Type
 	}
+
+	if ref, err := ParseRef(fieldType.String()); err == nil {
+		if ref.Module != "" && ref.Module != module.Name {
+			return // ignores references to other modules.
+		}
+	}
+
 	bodySym := scopes.ResolveType(fieldType)
 	if bodySym == nil {
-		merr = append(merr, errorf(resp, "ingress verb %s: couldn't resolve %s body type %s", n.Name, reqOrResp, fieldType))
+		merr = append(merr, errorf(r, "ingress verb %s: couldn't resolve %s body type %s", n.Name, reqOrResp, fieldType))
```

Comment:
This fixes the issue where `initial deploy failed: 14:3-3: ingress verb getQuote:` is returned as an error. This was because of a typo that used `resp` instead of `r` for the `errorf(` call